data/src/scripts/skill_thieving/scripts/pickpocketing/pickpocket.rs2

```diff
@@ -115,8 +115,8 @@ if (add(%lastcombat, 8) > map_clock) {
     return;
 }
 
-if(%stunned > map_clock) {
-    return;
+while(%stunned > map_clock) {
+    p_delay(0);
```

Comment:
I think this delays 1 tick? So basically while we're stunned, keep waiting 1 tick